pkg/perf/monitor.go

```diff
@@ -0,0 +1,86 @@
+package perf
+
+import (
+	"context"
+	"time"
+
+	"github.com/go-co-op/gocron"
+	"github.com/gomodule/redigo/redis"
+
+	"github.com/pkg/errors"
+	"github.com/threefoldtech/zos/pkg/utils"
+)
+
+// PerformanceMonitor holds the module data
+type PerformanceMonitor struct {
+	scheduler *gocron.Scheduler
+	pool      redis.Pool
```

Comment:
```suggestion
	pool      *redis.Pool
```
check the other comment to see why